docs/source/compilation/index.rst

```diff
@@ -0,0 +1,23 @@
+.. _compilation:
+
+Compilation
+=============
+
+Before starting Indico compilation, this guide assumes you've previously
+:ref:`cloned the development version <cloning>`.
+The first step is to generate a local distribution archive. Navigate to the Indico source folder
+(by default, it is ``~/dev/indico/src``) and run the following commands::
+
+    ./bin/maintenance/build-wheel.py --target-dir dist indico --add-version-suffix
```

Comment:
`--target-dir dist` can be omitted, it's the default anyway and people pretty much never need to override it.